jenkins/ci.suse.de/templates/cloud-mkcloud-job-ha-linuxbridge-template.yaml

```diff
@@ -25,6 +25,6 @@
             mkcloudtarget=all
             networkingplugin=linuxbridge
             networkingmode=vxlan
-            want_node_aliases=controller=2,compute=2
+            want_node_aliases=controller=3,compute=2
```

Comment:
I think this should be `{nodenumber_controller}`.

But than we also have to fix `nodenumber` and make it different for Cloud7 and 8

Reply:
@jsuchome  Ahh true I forget about that change will update.